cdap-api-common/src/main/java/io/cdap/cdap/api/data/schema/Schema.java

```diff
@@ -121,7 +121,8 @@ public boolean isSimpleType() {
     TIMESTAMP_MICROS(Type.LONG, "timestamp-micros"),
     TIME_MILLIS(Type.INT, "time-millis"),
     TIME_MICROS(Type.LONG, "time-micros"),
-    DECIMAL(Type.BYTES, "decimal");
+    DECIMAL(Type.BYTES, "decimal"),
+    DATETIME(Type.STRING, "datetime");
```

Comment:
It is stored as string is it because we want to preserve nanoseconds?

Reply:
Chose string as it will require less conversions for file based sinks and BQ sink. Discussed the pros and cons of string vs long here - https://docs.google.com/document/d/1x9Hrq5ie5v3SGM0u_2H7b018Z2-GkNY7V1fKzopBiAo/edit#heading=h.e6q5rm1dlkny .